Quadriga/src/main/webapp/WEB-INF/views/sites.jsp

```diff
@@ -0,0 +1,12 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib prefix="sec"
+	uri="http://www.springframework.org/security/tags"%>
+
+<!-- Content -->
+
+<article class="is-page-content">
+
+<span class="byline">Page Under Construction..Visit again to know us...</span>
```

Comment:
Otherwise let's actually create the page and fill it with dummy data (some dummy projects).